doc/whats_new/v0.23.rst

```diff
@@ -67,8 +67,16 @@ Changelog
   `ValueError` for arguments `n_classes < 1` OR `length < 1`.
   :pr:`16006` by :user:`Rushabh Vasani <rushabh-v>`.
 
+:mod:`sklearn.decomposition`
+............................
+
+- |Fix| :class:`decomposition.PCA` with a float `n_components` parameter, now
+   currently exclusively chooses the components that explains the variance
```

Comment:
```suggestion
   exclusively choose the components that explain the variance
```

Reply:
In my view, the edits look great! Thank you!